dali/base/dautils.cpp

```diff
@@ -1998,7 +2013,21 @@ class CTimedCache
         if (!item)
             return 0;
         CriticalBlock block(sect);
-        item->due = msTick()+PAGE_CACHE_TIMEOUT;
+        if ((maxPageCacheItems > 0) && (maxPageCacheItems == items.length()))
```

Comment:
We should default to something sensible for maxPageCacheItems both in impl. and in config.

I think 0 should mean cache off (which could be implemented by add throwing away)